web_src/less/chroma/dark.less

```diff
@@ -1,67 +1,70 @@
 .chroma .hl { background-color: #3f424d; } /* LineHighlight */
```

Comment:
I'm a bit confused: What is this file sorted by?
It seems pretty random where what is located.
Should we perhaps sort it alphabetically?

Reply:
Random, I guess. Ideal sort would be by comment but I can't do that automatically, but alphabetical seems better than current.

Reply:
Sorted both files.